From 632045d5c6298f9dcb289398fb35d5c0d2bb8bd8 Mon Sep 17 00:00:00 2001
From: Yasufumi Ogawa <ogawa.yasufumi@lab.ntt.co.jp>
Date: Wed, 3 Apr 2019 19:44:58 +0000
Subject: [PATCH 3/3] spp_vf: change inline func calls static inside
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Change `extern inline` funcitons to static to avoid compile error like
as following in some environments.

    error: ‘rte_eth_tx_burst’ is static but used in inline function
    ‘spp_eth_tx_burst’ which is not static

In this case, static rte_eth_tx_burst() should not be called from inline
function.

Signed-off-by: Yasufumi Ogawa <ogawa.yasufumi@lab.ntt.co.jp>
---
 src/vf/common/spp_port.c | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/src/vf/common/spp_port.c b/src/vf/common/spp_port.c
index fd76d79..06422c3 100644
--- a/src/vf/common/spp_port.c
+++ b/src/vf/common/spp_port.c
@@ -57,7 +57,7 @@ spp_port_ability_init(void)
 }
 
 /* Get information of port ability. */
-extern inline void
+void
 spp_port_ability_get_info(
 		int port_id, enum spp_port_rxtx rxtx,
 		struct spp_port_ability **info)
@@ -365,7 +365,7 @@ port_ability_each_operation(uint16_t port_id,
 }
 
 /* Wrapper function for rte_eth_rx_burst(). */
-extern inline uint16_t
+uint16_t
 spp_eth_rx_burst(
 		uint16_t port_id, uint16_t queue_id  __attribute__ ((unused)),
 		struct rte_mbuf **rx_pkts, const uint16_t nb_pkts)
@@ -387,7 +387,7 @@ spp_eth_rx_burst(
 }
 
 /* Wrapper function for rte_eth_tx_burst(). */
-extern inline uint16_t
+uint16_t
 spp_eth_tx_burst(
 		uint16_t port_id, uint16_t queue_id  __attribute__ ((unused)),
 		struct rte_mbuf **tx_pkts, uint16_t nb_pkts)
-- 
2.7.4

